### PR TITLE
Update per Phil

### PR DESCRIPTION
**Pull request by trevars**
https://cdis.slack.com/archives/CARERJJDV/p1566856111004400

### CODE REVIEW

File: ibdgc.datacommons.io/etlMapping.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ mappings:
   - name: ibdgc_etl
     doc_type: case
     type: aggregator
-    root: case
+    root: particpant
     props:
```
Comment (abgeorge7) [inline ibdgc.datacommons.io/etlMapping.yaml]: In the dictionary looks like it’s “case”, the front end text just says participant 
So need to change to case

File: ibdgc.datacommons.io/etlMapping.yaml [hunk 2]
```diff
@@ -11,6 +11,13 @@ mappings:
         props:
           - name: gender
           - name: race
+          - name: jewish
+          - name: hispanic
+      - path: diagnosis
```
Comment (abgeorge7) [inline ibdgc.datacommons.io/etlMapping.yaml]: “diagnoses”

File: ibdgc.datacommons.io/portal/gitops.json [hunk 3]
```diff
@@ -119,7 +119,12 @@
           "fields":[
             "project_id",
             "gender",
-            "race"
+            "race",
+            "jewish",
+            "hispanic",
+            "ibd_affection_status",
+            "disanosis",
```
Comment (abgeorge7) [inline ibdgc.datacommons.io/portal/gitops.json]: “diagnosis”

File: ibdgc.datacommons.io/portal/gitops.json [hunk 4]
```diff
@@ -130,6 +135,11 @@
         "project_id",
         "gender",
         "race",
+        "jewish",
+        "hispanic",
+        "ibd_affection_status",
+        "disanosis",
```
Comment (abgeorge7) [inline ibdgc.datacommons.io/portal/gitops.json]: “diagnosis”

File: ibdgc.datacommons.io/etlMapping.yaml (hunk 1 above)
Comment (trevars) [reply ibdgc.datacommons.io/etlMapping.yaml]: Check

File: ibdgc.datacommons.io/etlMapping.yaml [hunk 5]
```diff
@@ -11,6 +11,13 @@ mappings:
         props:
           - name: gender
           - name: race
+          - name: jewish
+          - name: hispanic
+      - path: diagnoses
+        props:
+          - name: ibd_affection_status
+          - name: diagnoses
```
Comment (mfshao) [inline ibdgc.datacommons.io/etlMapping.yaml]:
```suggestion
          - name: diagnosis
```

**Pull request merged**